### PR TITLE
add new study: Burrows_2020

### PR DESCRIPTION
**Pull request by ehwenk**
* large data set documenting different fire response traits for WA taxa
* compiled by Neil Burrows and originally available on NatureMap, but data acquired through Carl Gosper.

### CODE REVIEW

**Review by yangsophieee — CHANGES_REQUESTED** [review]
Everything looks great! I just flagged some definition issues for the new traits.

File: config/traits.yml [hunk 1]
```diff
@@ -2335,7 +2335,21 @@ traits:
...
+      type: numeric
+      units: mo
+      label: Time from first flowering to decline in flowering 
+      allowed_values_min: 1
+      allowed_values_max: 1000    	
+    reproductive_maturity_time_to_peak:
+      description: The time interval from XXX
+      type: numeric
+      units: mo
+      label: Time from first flowering to peak flowering
```
[the remark this answers is not in the document]
Comment (ehwenk) [reply config/traits.yml:2350]: I changed them and added proper definitions - just needed to do it first on APD files

File: config/traits.yml [hunk 2]
```diff
@@ -2335,7 +2335,21 @@ traits:
       type: categorical
       label: Fruiting time, by month
       comments: Trait keyed as a sequences of 12 Ns (not flowering) and Ys (flowering),
-        representing the 12 months, starting with January.
+        representing the 12 months, starting with January.    	
+    reproductive_maturity_time_to_decline:
+      description: The time interval from XXX
+      type: numeric
+      units: mo
+      label: Time from first flowering to decline in flowering
```
[the remark this answers is not in the document]
Comment (ehwenk) [reply config/traits.yml:2343]: Yup, fixed.

File: config/traits.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (yangsophieee) [reply config/traits.yml:2350]: The label still says "from first flowering to peak flowering"

File: config/traits.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (yangsophieee) [reply config/traits.yml:2343]: Description looks great! Label still needs to be changed

[the remark this answers is not in the document]
Comment (yangsophieee) [reply config/traits.yml]: I've changed the label.

File: config/traits.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (yangsophieee) [reply config/traits.yml]: I've changed the label.

**Review by yangsophieee — APPROVED** [review]

**Pull request merged**